### ISSUE TITLE
Why does it load screenshots from 2 different versions of HTTP protocol?

### ISSUE THREAD

**Issue by PackmanDude** [MinersStudios/WMSite#6]
```
GET
https://whomine.net/assets/img/screenshots/10.webp
[HTTP/3 200 OK 434ms]

GET
https://whomine.net/assets/img/screenshots/10.webp
[HTTP/1.1 200 OK 0ms]


```

**Comment by p0loskun** [comment]
This happened when the _getRandomScreenshot_ function set src for the local image and the element on the page. 
Second time is loading from the cache

`
    setTimeout(() => {
        screenshotImg.style.backgroundImage = `url(${image.src})` _//sets src for element on the page after 350 ms (loaded from cache)_
    }, 350)

    isScreenshotLoading = true

    let image = new Image()
    image.src = `./assets/img/screenshots/${screenshots[index]}.webp` _//set src for local image object (runs first)_
    image.onload = () => {
        randomButton.classList.toggle('dark', getCachedBrightness(image.src))

        setTimeout(() => {
            screenshotImg.classList.add('loaded')
            screenshotImg.style.opacity = '1'
        }, 350)

        setTimeout(() => {
            isScreenshotLoading = false
        }, 500)
    }
`

**Comment by PackmanDude** [comment]
I mean, what did you change. I can probably tell that there was no such thing after 40bc6ccd4502630b8c4e3de2b451d903f732a5fb, but still, loading from cash is fine, and looking at it now, Web API is not that well-made to keep 1 load, or at least I think so now.

**Comment by PackmanDude** [comment]
I mean, what did you change. I can probably tell that there was no such thing after 40bc6ccd4502630b8c4e3de2b451d903f732a5fb.

**Comment by p0loskun** [comment]
The problem you are referring to started occurring when the random screenshot section was created and was only fixed in 659c731. After that change, the div tag with the background image that was set via the style property, which caused the image to be double loaded when the getRandomScreenshot function was called, was replaced with an img tag, allowing the image object to be associated with the element.

**Comment by PackmanDude** [comment]
Nice.
